models/studentCourseSchema.js

```diff
@@ -0,0 +1,12 @@
+var mongoose = require('mongoose');
+var Schema = mongoose.Schema;
+
+// Create student-course instance Schema
+var studentCourseSchema = mongoose.Schema({
+  studentId: Schema.Types.ObjectId,
+  courseId: Schema.Types.ObjectId,
+  semester: String,
+  grade: String
+});
+
+module.exports = mongoose.model("StudentCourse", studentCourseSchema);
```

Comment:
Github didn't catch that my new commit obsoleted this comment, but it did. Done.